packages/wrangler/src/experimental-flags.ts

```diff
@@ -2,7 +2,8 @@ import { AsyncLocalStorage } from "async_hooks";
 import { logger } from "./logger";
 
 type ExperimentalFlags = {
-	FILE_BASED_REGISTRY: boolean;
+	FILE_BASED_REGISTRY?: boolean;
+	RESOURCES_PROVISION?: boolean;
```

Comment:
Was there a reason to make these optional? Since it's an internal API I feel like it'd be better to keep these required?

Reply:
Mostly to avoid the need to update the feature flag on every place we run an async context. If you prefer making it explicit, I can change that.

Reply:
https://github.com/cloudflare/workers-sdk/pull/7348/commits/ab67e0dc192c0c1062fff4380c9d6fad16ac8aec